tests/install_upgrade_operators/hco_enablement_golden_image_updates/multiarch/conftest.py

```
@@ -0,0 +1,119 @@
+
+@pytest.fixture(scope="session")
+def worker_architectures(schedulable_nodes):
+    return {node.labels[KUBERNETES_ARCH_LABEL] for node in schedulable_nodes}
+
+
+@pytest.fixture(scope="class")
+def control_plane_architecture(control_plane_nodes):
+    return get_control_plane_architecture(control_plane_nodes=control_plane_nodes)
+
+
+@pytest.fixture()
```

Ruff (0.15.13)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)